cdap-cli/src/main/java/co/cask/cdap/cli/CLIConfig.java

```diff
@@ -77,6 +80,27 @@ public Integer get() {
     }
   };
 
+  /*
+   * Wrapper class for reading/writing of username + accessToken
+   */
+  private class UserAccessToken {
```

Comment:
After reading the code again, why not just change the "AccessToken" class to include username (maybe as a nullable final field) as well?